scenarios/cpp/windows/captioning/caption.cpp

```diff
@@ -0,0 +1,580 @@
+/* NOTES:
+- In Windows, place the following files from the Speech SDK in the same folder as your compiled .exe:
+	- Microsoft.CognitiveServices.Speech.core.dll
+	- Microsoft.CognitiveServices.Speech.extension.audio.sys.dll
+*/
+
+/*
+Dependencies
+*/
+
+// exception
+#include <exception>
```

Comment:
Can you put the comment at the end of the #include, to make it more compact? I think that's what we do elsewhere. Should be:
#include <exception> // exception